### DISCUSSION TITLE
Model.cpkt self-made option?

### DISCUSSION THREAD

**Discussion by ZeroHackz** [lkwq007/stablediffusion-infinity#57] [Q&A]
Hi all, 

I'm reading through the code, and it seems that it uses the Hugging Face Models through fetching with your tokens.
I wonder If I could modify the code so I could reference my own model.cpkt file?

Is there a self made model option? Or should I just edit the "get_model()" function in app.py locally and return my model os path?

Thanks in advance, 
Have a nice weekend.

**Comment by lkwq007** [comment]
For you own model checkpoint, you may need to convert it into diffusers compatible model weights using https://github.com/huggingface/diffusers/blob/main/scripts/convert_original_stable_diffusion_to_diffusers.py
and then modify `model_name` to you local path to the model weight folder
https://github.com/lkwq007/stablediffusion-infinity/blob/57fff11bd8db07a4d836d97de37b3853be52e951/app.py#L121
